torch/distributed/algorithms/model_averaging/averagers.py

```diff
@@ -101,13 +101,28 @@ def __init__(
 
     def average_parameters(self, params):
         r"""
-        Averages parameters if ``step`` is no less than ``warmup_steps``
+        Averages parameters or parameter groups of an optimizer if ``step`` is no less than ``warmup_steps``
         and it can be divided by ``period``, where ``step`` is increased by 1
         at each iteration in the training loop.
+        params: average model.parameters() or parameter groups of an optimizer
```

Comment:
This doc is a bit unclear, the pattern we normally follow is such as:

"""
# ... Rest of doc
Args:
   params: Union[Iterable[torch.nn.Parameter], Iterable[Dict[str, torch.nn.Parameter]]]: List of parameters or parameter groups of an optimizer.
"""

Reply:
fix